code/lang/packs/English.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Can't we get rid of these outer brackets, they are really annoying and could lead to code inconsistencies and issues in contributions (tab/space indentation).

Then it would be the perfect system, as it was planned to be 😃

Reply:
Not sure if I'm understanding you totally correctly Alf, but I'm pretty sure those are required within JSON as curly brackets are the 'object' markers in order for the parser to read it. Could be mistaken.

Reply:
Yea yea, for sure (even that there are different possibilities out there), but to simplify the creation, we could easily add brackets at start and end before reading the data ^^

Anyways, if we wanna add configs to the json as well, then these brackets are needed tho

Reply:
Reasonable. Could alternatively swap out Json for just reading line by line and parsing the individual lines through our own code. Might be a tad bit faster than the Json reader.

I'd support having the file reader check to see if the brackets exist at the start and end within the file, then add them if they do not, so it works whether you have them or not.